ubuntu/ubuntu-18.x/ubuntu-18.04-hpc/hpc-tuning.sh

```diff
@@ -29,12 +27,19 @@ EOF
 echo "vm.zone_reclaim_mode = 1" >> /etc/sysctl.conf
 sysctl -p
 
+# Install WALinuxAgent
+apt-get install python3-setuptools
+git clone https://github.com/longlimsft/WALinuxAgent.git
```

Comment:
This will break if github.com/longlimsft/WALinuxAgent.git is updated. PLease use the main repository and cherry-pick the required commit.